public/data/VarunNarayanan.json

```diff
@@ -0,0 +1,23 @@
+{
+   "name":"Varun Narayanan",
+   "type":"personal",
+   "bio":"Open Source Enthusiast! And a Web Developer too",
+   "avatar":"https://github.com/Varunmnx.png",
+   "links":[
+      {
+         "name":"Follow me on GitHub",
+         "url":"https://github.com/Varunmnx",
+         "icon":"github"
+      },
+      {
+         "name":"Follow me on Twitter",
+         "url":"https://twitter.com/Varun_Narayana1",
+         "icon":"twitter"
+      },
+      { "name": "Connect with me on LinkedIn", 
+        "url":"https://www.linkedin.com/in/varun-narayanan-74b653193", 
+        "icon": "linkedin" 
+      },
```

Comment:
Please remove this Trailing comma from here! 
```suggestion
      }
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix